Prefabs/Interactions/Interactables/SharedResources/Scripts/Grab/GrabInteractableInternalSetup.cs.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: dd6d3bd046e0d7043aaed76be77f521b
+guid: 8314c9a77122ba744ac0a55af02c58b4
```

Comment:
Was this on purpose? This GUID change is probably making existing usages in Unity assets break this link to script?

Reply:
The break was caused by a rebase where the script moved locations between the commits (and was actually a git move so it just did a rename)

but it seems that Unity got itself all confused and thought the guid had changed. Fixing this is a reflog job so it's probably best to just push ahead and ignore this ever happened!